app/errors/payments.rb

```diff
@@ -0,0 +1,10 @@
+module Payments
+  class PaymentsError < StandardError
```

Comment:
Using custom exceptions to avoid exposing Stripe's to the controller. In case in the future we'll have more than one payment system but also generally to separate responsibilities